### PR TITLE
Database config file and table creation file added

### PR DESCRIPTION
**Pull request by mak-skid**
I added a config file `dbsetting.py` and an ORM file `event.py`. I needed to `git reset --hard` to revert the commits that contained sensitive information.

### CODE REVIEW

**Review by IPINGCHOU — COMMENTED** [review]
you don't have to commit __pycache__, you may add it in `.gitignore`

File: dbsetting.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from sqlalchemy.ext.declarative import declarative_base
+
+host= os.getenv("HOST"),
+user=os.getenv("USERNAME"),
+passwd= os.getenv("PASSWORD"),
+db= os.getenv("DATABASE"),
+ssl_mode = "VERIFY_IDENTITY",
+ssl      = {
+    "ca": "/etc/ssl/cert.pem"
+}
```
Comment (IPINGCHOU) [inline dbsetting.py]: global vars are recommended to name with capitals, like `HOST`

File: event.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    __table_args__=({"mysql_engine": "InnoDB"})
+    time = Column("time", DateTime, nullable=False, primary_key=True)
+    area = Column("area", Text, nullable=False)
+    cause = Column("cause", Text, nullable=False)
+    direction = Column("direction", Text, nullable=False, primary_key=True)
+    section = Column("section", Text, nullable=False, primary_key=True)
+    route = Column("route", Text, nullable=False)
+    detail = Column("detail", Text, nullable=False)
+    length = Column("length", Integer, nullable=True)
+    coordinates = Column("coordinates", Text, nullable=True)
```
Comment (IPINGCHOU) [inline event.py]: These should be defined under the function `def init(self)`, as a part of the initiation of the class

Comment (mak-skid) [reply event.py]: Ok, but what is the downside of not using `def init(self)`? I googled tutorials and they don't `__init__` in the class.

**Review by IPINGCHOU — APPROVED** [review]
LGTM

File: event.py (hunk 2 above)
Comment (IPINGCHOU) [reply event.py]: class object will run everything defined in `__init__` when the var which uses the class get declared. So anything you need for a class to be pre-defined before running any functionalities, I recommend to define it in `__init__`

Comment (mak-skid) [reply event.py]: Makes sense !

**Pull request merged**